modules/installation-vsphere-machines.adoc

```diff
@@ -34,20 +34,14 @@ ifndef::openshift-origin[]
 {
   "ignition": {
     "config": {
-      "append": [
+      "merge": [
         {
           "source": "<bootstrap_ignition_config_url>", <1>
-          "verification": {}
         }
       ]
     },
-    "timeouts": {},
-    "version": "2.1.0"
+    "version": "3.1.0"
   },
```

Comment:
Same question about the comma

Reply:
@jianlinliu should I remove the comma at the end of [this line](https://github.com/openshift/openshift-docs/pull/24805/files#diff-345226365a5ceae78307a593f245c07dR39)? It doesn't appear necessary any longer. This was taken from the vSphere UPI doc [here](https://github.com/openshift/installer/blob/master/docs/user/vsphere/install_upi.md#bootstrap-vm), which may need to also be updated.

Reply:
@jianlinliu can you confirm our suspicions on the erroneous comma in this vSphere YAML file?

Reply:
Yeah, the ending comma at line 39 and line 44 should be removed. Here a online json/yaml converter - https://www.json2yaml.com/, that can help you confirm format issue.